resources/views/resultplot.blade.php

```diff
@@ -79,25 +69,25 @@
                 <thead>
 
                     <tr>
-                        <th class="mobile-hide">No.</th>
-                        <th class="mobile-hide">SHIFT</th>
-                        <th>Tanggal</th>
-                        <th>Waktu</th>
-                        <th  class="mobile-hide">Kuota</th>
-                        <th>LIST</th>
+                        <th id="table-backgrounds" class="table-content-custom mobile-hide">No.</th>
+                        <th id="table-backgrounds" class="table-content-custom mobile-hide">SHIFT</th>
+                        <th id="table-backgrounds" class="table-content-custom">Tanggal</th>
+                        <th id="table-backgrounds" class="table-content-custom">Waktu</th>
+                        <th id="table-backgrounds" class="table-content-custom mobile-hide">Kuota</th>
+                        <th id="table-backgrounds" class="table-content-custom">LIST</th>
```

Comment:
```suggestion
                        <th class="table-content-custom mobile-hide">No.</th>
                        <th class="table-content-custom mobile-hide">SHIFT</th>
                        <th class="table-content-custom">Tanggal</th>
                        <th class="table-content-custom">Waktu</th>
                        <th class="table-content-custom mobile-hide">Kuota</th>
                        <th class="table-content-custom">LIST</th>
```

styling yang ada di id tadi satuin aja ke `table-content-custom`

Reply:
gabisa bang, untuk background nya ketimpa sm file lain, makanya itu juga udah dibuat alternatifnya